docs/architecture/adr-050-sign-mode-textual-annex1.md

```diff
@@ -0,0 +1,89 @@
+# ADR 049: SIGN_MODE_TEXTUAL - Annex 1 Value Renderers
+
+## Changelog
+
+- Dec 06, 2021: Initial Draft
+
+## Status
+
+Draft
+
+## Abstract
+
+This Annex describes value renderers, which are used for displaying values in a human-friendly way.
+
+## Value Renderers
+
+Value Renderers describe how values of different Protobuf types should be automatically rendered. Value renderers can be formalized as a set of bijective functions `func renderT(value T) string`, where `T` is one of the below Protobuf types on which this spec is defined.
+
+### Protobuf `number`
+
+- Applies to numeric integer types (`uint64`, etc.) and their casted types (e.g. in Golang: `sdk.Dec`, `sdk.Int`).
+- Formatting with `'`s for every three integral digits.
+- Usage of `.` to denote the decimal delimitier.
+
+### Examples
+
+- sdk.Int, integers: `1000` -> `1'000`
+- sdk.Dec; `1000000.00` -> `1'000'000.00`
```

Comment:
How many fractional digits should be shown here? `sdk.Dec` is a fixed point type with 18 fractionals, right?

Reply:
As discussed on the call, let's do:
- never truncate
- remove trailing zeroes